apps/careers-app/backend/auth.py

```
@@ -0,0 +1,101 @@
+            resp.raise_for_status()
+            data = resp.json()
+
+        if not data.get("active"):
+            raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail="Token inactive")
+        sub = data.get("sub", "")
+        if not sub:
+            raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail="Invalid token")
+        return {"sub": sub}
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)